Technical review: Document customizable select listboxes (#43352)
mdn/content: files/en-us/web/html/reference/elements/select/index.md

## Styling with CSS

> The `<select>` element has historically been notoriously difficult to style productively with CSS, hence features being introduced to enable creating fully customizable select elements. See the following guides for more information:

Reviewer: ```suggestion
The `<select>` element has historically been difficult to style effectively with CSS.
The following guides have information about newer features that have been introduced to enable fully customizable select elements:
```
